src/lang/en.json

```diff
@@ -16,6 +16,7 @@
   "analysis_expected": "of",
 
   "app_name": "OpenDota",
+  "app_localization": "Localization",
```

Comment:
I think it should go in the file so the translators can modify it.  We can then pull it out of each file by language code?